src/repositories/user/UserRepository.ts

```diff
@@ -5,6 +5,9 @@ import IUserModel from './IUserModel';
 import { idText } from 'typescript';
 
 export default class UserRepository {
+    static count() {
+        throw new Error('Method not implemented.');
+    }
```

Comment:
Implement this function to count the document in the user collection using `userModel.countDocuments()`